lib/tasks/seed.rake

```diff
@@ -1,13 +1,13 @@
-require 'faker'
+require 'faker/user'
 
 namespace :seed do
   desc "Create users with data from ffaker."
   task users: :environment do
-    faker = Faker.new(count)
-    faker.create_users
+    faker = Faker::Users.new(count)
+    faker.create!
   end
 end
 
 def count
```

Comment:
We should pass in a symbol for the task, so it returns the right environment variable, or just name this `count_users` for now.
E.g.
```
def count(task_name)
  ENV.fetch("NUM_#{task_name.upcase}", default(task_name)
end

def default(task_name)
  case task_name
  when :users
    50
  ...
end
```
Or we could just stick with a single case statement
```
def count(task_name)
  case task_name
  when :users
    ENV.fetch("NUM_USERS", 50
  ...
end
```